docs/glossary.md

```diff
@@ -0,0 +1,314 @@
+account key. This term is specifically used to describe accounts are auto-created by doing a
+CryptoTransfer to an alias that is of EVM address size, which allows applications to create user
+accounts instantly, even without an internet connection, by assigning an account alias. See
+also [Account Aliases](#account-aliases), https://hips.hedera.com/hip/hip-32
+and https://hips.hedera.com/hip/hip-583.
+
+---
+
+## Leaky Tests
+
+**Leaky Tests**: Are unit or other tests that should not be run in parallel because they can “leak”
+some of their state or data out into the global state.
+
+---
+
+## Precompiles
+
+**Precompiles**: Behave as if they were contracts deployed at specific addresses (with one
+exception: The EIP-4788 "beacon roots contract" which is a real contract deployed at a specific
+address) In Ethereum, they are very low addresses, 0x00 .. 0x0F).
+They are not implemented as contracts because that would be far too slow, instead they are built-in
+to the code of the Ethereum EVM, which checks to see if the address you're calling is one of the
+precompiles and then just runs the related code.
+We call our equivalent [System Contracts](#system-contracts) but there may still some legacy
+references to "precompiles" in the codebase and older documentation.
+
+## Privileged System Accounts
+
+**Privileged System Accounts**: An account with an entity ID of 0.0.1000 or lower. These are
+accounts that are authorized to handle [Admin Transactions](#admin-transaction).
+Some privileged accounts, like account 0.0.2 or 0.0.58, are owned by the Hedera Council. This means
+their keys are on the account and their signatures would be required if the account is being
+modified or used to pay for - [Admin Transactions](#admin-transaction)
+
+---
+
+## Security Model V2
+
+**Security Model v2**: Hedera introduced the HSCS Security Model v2 to enhance the security of its
+network building with improved features to better safeguard against potential vulnerabilities and
+attacks. By incorporating enhanced cryptographic techniques, robust consensus mechanisms,
+decentralized governance, and continuous monitoring.
+Key features include - Restrictions on Smart contracts Storage; Prohibition on Delegate Calls to
+System Smart Contracts; Limited Access to EOAs' Storage; Token Allowance Requirement for Balance
+Changes.
+Refer to
+this [official blog post](https://hedera.com/blog/hedera-smart-contract-service-security-model-update)
+for more information.
+
+## Sidecars
+
+**Sidecars**: In Services, a sidecar refers to additional records that are created alongside the
+main transaction records. These sidecar records provide more detailed information about the
+transactions, making it easier to debug and understand the state changes that occur as a result of
+the transactions. Note that these are related to the current record stream implementation and will
+be replaced by block streams in the future.
+
+## Synthetic Transaction
```

Comment:
Unfortunately my previous comment to you was wrong.  This definition is true but is partial, it turns out that all "Child Transactions" are considered "Synthetic Transactions" too.